handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: drush clear cache
```

Comment:
Good point, and an easy fix.  I prefer `drush cache clear drush`:  except for lack of a hyphen, that it what I would use on the command line.  My second choice would be closer to an English sentence, like `Clear the drush cache`.